hack/manifest-gen/assets/charts/eck/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Core configuration values
+operator:
+  # Name of the operator installation.
+  name: elastic-operator
+  # Namespace to install the operator to.
+  namespace: elastic-system
+  # Version of the operator. Should be set to the desired container image tag.
+  version: 1.2.0
```

Comment:
Could we not reuse `{{ .Chart.AppVersion }}` ?

Reply:
That makes it tricky to test unreleased images. With an explicit setting, I can test a locally built snapshot by passing something like `--set=operator.version=1.2.0-48b3bc4e --set=operator.image.repository=docker.elastic.co/eck-dev/eck-operator-cell`

Reply:
I see there's this related issue about allowing to set appversion without editing the chart yaml https://github.com/helm/helm/issues/8194